storybook/src/admin-rte/RteCustomInlineStyles.stories.tsx

```diff
@@ -10,7 +10,9 @@ const rteOptions: IRteOptions = {
     customInlineStyles: {
         HIGHLIGHT: {
             label: "Highlight!",
-            icon: Favorite,
+            icon: () => {
+                return <Favorite />;
+            },
```

Comment:
Do we need to fix this in the RTE?

Reply:
I think we do not need to fix this in rte, in my opinion it is fine like its implemented here.

Reply:
I disagree. This error was probably caused by #3231 and should be fixed separately. There's a reason why we chose to use SvgIcon as type, e.g., to be able to pass color and disabled from inside the RTE.

Reply:
Fixed it here: https://github.com/vivid-planet/comet/pull/3299